lib/external/shapelib/dbfopen.c

```diff
@@ -34,7 +34,7 @@
  * DEALINGS IN THE SOFTWARE.
  ******************************************************************************/
 
-#include "shapefil.h"
+#include <grass/shapefil.h>
```

Comment:
`#include "shapefil.h"` should be fine because this header is right here?

Reply:
You're right, I just mechanically reapplied the old patch in this regard. Now, the patches needed is really at minimum.

Updated accordingly.